src/styles/book.css

```diff
@@ -0,0 +1,121 @@
+.d-flex {
+  display: flex;
+}
+
+.book-container {
+  font-family: 'Roboto Slab', serif;
+  justify-content: space-between;
+  align-items: center;
+  border-top: solid 1px rgb(214, 214, 214);
+  margin-bottom: 5px;
+  padding: 10px 5%;
+}
```

Comment:
- [ ] Please let's ensure that the book container has a white `background-color` and light grey `border` not forgetting about the `border-radius`. Styles we were supposed to have picked from the provided design. See screenshot below:

  ![image](https://user-images.githubusercontent.com/45002654/150168464-8e424e03-46f7-4afc-a0a1-c0a526a946ef.png)

Reply:
done